mbs-s0015-20240504/check_spectra_mpi.py

```diff
@@ -0,0 +1,128 @@
+import os
+
+import numpy as np
+from mpi4py import MPI
+from pixell import enmap, curvedsky, mpi as p_mpi
+from mnms import utils
+
+opj = os.path.join
+comm = MPI.COMM_WORLD
+
+imapdir = '/mnt/home/aduivenvoorden/project/so/20240504_mss0002_lat/renamed'
+simdir = '/mnt/home/aduivenvoorden/project/actpol/maps/mnms/sims'
```

Comment:
are you reading ACT data here? so people need to have access to ACT to run this or are they public?

Reply:
This is a path the CCA cluster, I added the corresponding NERSC path as a comment.